deployment/docker-compose/docker-compose.dev.yml

```diff
@@ -13,6 +14,12 @@ services:
       - db_volume:/var/lib/postgresql/data
     ports:
       - 5434:5432
+  redis:
```

Comment:
Why do we add this to `docker-compose.dev` and not to `docker-compose`. We'll need it all instances right?